airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryDestination.java

```diff
@@ -269,7 +270,7 @@ public void acceptTracked(AirbyteMessage message) {
         final JsonNode data = Jsons.jsonNode(ImmutableMap.of(
             COLUMN_AB_ID, UUID.randomUUID().toString(),
             COLUMN_DATA, Jsons.serialize(message.getRecord().getData()),
-            COLUMN_EMITTED_AT, message.getRecord().getEmittedAt()));
+            COLUMN_EMITTED_AT, Instants.toSeconds(message.getRecord().getEmittedAt())));
```

Comment:
You're positive it is seconds and not milliseconds ?

Reply:
sadface. Turns out BQ actually uses _microseconds_. https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#timestamp_type. I was confused because writing seconds produced the correct "human-formatted" timestamp on BQ when inspected by eye (like the screenshot below). 
<img width="168" alt="Screen Shot 2020-11-14 at 6 47 34 PM" src="https://user-images.githubusercontent.com/6246757/99161716-df24ee00-26a9-11eb-80f3-1601ac7e374d.png">

Goes to show the first thing I should have looked at was the docs before making changes. I changed this to use the official google libraries for date formatting and output as a timestamp-formatted string.